mule-extensions-api/src/main/java/org/mule/runtime/extension/api/loader/DeclarationEnricherPhase.java

```diff
@@ -40,6 +37,12 @@
    */
   STRUCTURE,
 
+  /**
+   * This phase contains all {@link DeclarationEnricher}s that ensure consistency among elements, required by enrichers in the
```

Comment:
add a brief comment indicating what kind of things should this phase be used for